.buildkite/winlogbeat/pipeline.winlogbeat.yml

```diff
@@ -2,39 +2,152 @@
 name: "beats-winlogbeat"
 
 env:
-  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
-  IMAGE_WIN_10: "family/general-windows-10"
-  IMAGE_WIN_11: "family/general-windows-11"
-  IMAGE_WIN_2016: "family/core-windows-2016"
-  IMAGE_WIN_2019: "family/core-windows-2019"
-  IMAGE_WIN_2022: "family/core-windows-2022"
   GCP_DEFAULT_MACHINE_TYPE: "c2d-highcpu-8"
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
   GCP_WIN_MACHINE_TYPE: "n2-standard-8"
-  BEATS_PROJECT_NAME: "winlogbeat"
+
+  IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
+  IMAGE_WIN_10: "family/platform-ingest-beats-windows-10"
+  IMAGE_WIN_11: "family/platform-ingest-beats-windows-11"
+  IMAGE_WIN_2016: "family/platform-ingest-beats-windows-2016"
+  IMAGE_WIN_2019: "family/platform-ingest-beats-windows-2019"
+  IMAGE_WIN_2022: "family/platform-ingest-beats-windows-2022"
+
+  # Other deps
+  ASDF_MAGE_VERSION: 1.15.0
 
 steps:
+  - group: "Winlogbeat Mandatory Tests"
+    key: "winlogbeat-mandatory-tests"
+
+    steps:
+      - label: ":ubuntu: Winlogbeat Crossccompile"
+        key: "mandatory-cross-compile"
+        command: "make -C winlogbeat crosscompile"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "winlogbeat/build/*.xml"
+          - "winlogbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "Winlogbeat: Crosscompile"
+
+      - label: ":windows: Winlogbeat Win-2016 Unit Tests"
+        command: |
+          Set-Location -Path winlogbeat
+          mage build unitTest
+        key: "mandatory-win-2016-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2016}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "winlogbeat/build/*.xml"
+          - "winlogbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "Winlogbeat: Win-2016 Unit Tests"
```

Comment:
according to https://github.com/elastic/beats/blob/52430734d9127907211e4bf0a333be6725bd84dd/winlogbeat/Jenkinsfile.yml#L33 shouldn't windows 2016 for winlogbeat#7.17 be also in the extended windows group, rather than in the mandatory tests group?